kuma/wiki/tests/test_forms.py

```diff
@@ -14,10 +15,10 @@ class FormEditorSafetyFilterTests(UserTestCase):
     def test_form_onload_attr_filter(self):
         """RevisionForm should strip out any harmful onload attributes from
         input markup"""
-        d, r = doc_rev("""
+        rev = revision(save=True, is_approved=True, content="""
```

Comment:
I saw the note that someone wants to get rid of `doc_rev` so I figured I'd help while in this file.